### ISSUE TITLE
Get started steps not working on Fedora 35

### ISSUE THREAD

**Issue by gabriel-farache** [project-flotta/project-flotta.github.io#12]
I followed the "Get Started" steps and it did not work properly:

- I had to manually run some gem commands given by the bundle exec command (I had already executed bundle install):
```
$ bundle exec jekyll server
Ignoring eventmachine-1.2.7 because its extensions are not built. Try: gem pristine eventmachine --version 1.2.7
Ignoring ffi-1.15.4 because its extensions are not built. Try: gem pristine ffi --version 1.15.4
Ignoring http_parser.rb-0.6.0 because its extensions are not built. Try: gem pristine http_parser.rb --version 0.6.0
Ignoring racc-1.6.0 because its extensions are not built. Try: gem pristine racc --version 1.6.0
Ignoring sassc-2.4.0 because its extensions are not built. Try: gem pristine sassc --version 2.4.0
Could not find sassc-2.4.0 in any of the sources
```
- Even after doing so I got the following error:
```
11:58 $ bundle exec jekyll server
Configuration file: /home/gabriel/git/flotta.github.io/_config.yml
            Source: /home/gabriel/git/flotta.github.io
       Destination: /home/gabriel/git/flotta.github.io/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
       Jekyll Feed: Generating feed for posts
                    done in 0.096 seconds.
 Auto-regeneration: enabled for '/home/gabriel/git/flotta.github.io'
                    ------------------------------------------------
      Jekyll 4.2.1   Please append `--trace` to the `serve` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
/usr/share/gems/gems/jekyll-4.2.1/lib/jekyll/commands/serve/servlet.rb:3:in `require': cannot load such file -- webrick (LoadError)
	from /usr/share/gems/gems/jekyll-4.2.1/lib/jekyll/commands/serve/servlet.rb:3:in `<top (required)>'
	from /usr/share/gems/gems/jekyll-4.2.1/lib/jekyll/commands/serve.rb:179:in `require_relative'
	from /usr/share/gems/gems/jekyll-4.2.1/lib/jekyll/commands/serve.rb:179:in `setup'
	from /usr/share/gems/gems/jekyll-4.2.1/lib/jekyll/commands/serve.rb:100:in `process'
	from /usr/share/gems/gems/jekyll-4.2.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail'
	from /usr/share/gems/gems/jekyll-4.2.1/lib/jekyll/command.rb:91:in `each'
	from /usr/share/gems/gems/jekyll-4.2.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail'
	from /usr/share/gems/gems/jekyll-4.2.1/lib/jekyll/commands/serve.rb:86:in `block (2 levels) in init_with_program'
	from /usr/share/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute'
	from /usr/share/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each'
	from /usr/share/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute'
	from /usr/share/gems/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go'
	from /usr/share/gems/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program'
	from /usr/share/gems/gems/jekyll-4.2.1/exe/jekyll:15:in `<top (required)>'
	from /usr/bin/jekyll:23:in `load'
	from /usr/bin/jekyll:23:in `<top (required)>'
	from /usr/share/gems/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:63:in `load'
	from /usr/share/gems/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:63:in `kernel_load'
	from /usr/share/gems/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:28:in `run'
	from /usr/share/gems/gems/bundler-2.2.22/lib/bundler/cli.rb:474:in `exec'
	from /usr/share/gems/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	from /usr/share/gems/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from /usr/share/gems/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	from /usr/share/gems/gems/bundler-2.2.22/lib/bundler/cli.rb:30:in `dispatch'
	from /usr/share/gems/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	from /usr/share/gems/gems/bundler-2.2.22/lib/bundler/cli.rb:24:in `start'
	from /usr/share/gems/gems/bundler-2.2.22/libexec/bundle:49:in `block in <top (required)>'
	from /usr/share/gems/gems/bundler-2.2.22/lib/bundler/friendly_errors.rb:128:in `with_friendly_errors'
	from /usr/share/gems/gems/bundler-2.2.22/libexec/bundle:37:in `<top (required)>'
	from /usr/bin/bundle:23:in `load'
	from /usr/bin/bundle:23:in `<main>'
```

Not being very familiar with ruby nor jekyll, I do not even know where to start looking.

**Comment by eloycoto** [comment]
Maybe we need to have a Makefile and docker-container here TBH.

**Comment by gabriel-farache** [comment]
Agree, building projects in container allow more flexibility and ensure the building will go smoothly no matter the platform.

**Comment by gabriel-farache** [comment]
Solved by PR #14
